megablocks/layers/dmlp_registry.py

```diff
@@ -0,0 +1,29 @@
+from typing import Union
+from megablocks.layers import mlp
+from megablocks.layers import glu
+from megablocks.layers.arguments import Arguments
+
+MlpType = Union[mlp.SparseMLP, glu.SparseGLU]
+
+class DMlpRegistry:
```

Comment:
I usually make the "d" lower case - dMoE, dMLP. And you can capitalize MLP, like we do on the class names.